metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -946,6 +950,9 @@ class MetalsLanguageServer(
         case Right(value) =>
           val old = userConfig
           userConfig = value
+          if (userConfig.excludedPackages != old.excludedPackages) {
+            slowConnectToBuildServer(forceImport = true).asJavaObject
```

Comment:
So I'm not 100% sure what the best thing to do here is. This works as it just reconnects and re-indexes. Then the workspace symbol search, completions, etc reflect he newly indexed result. However, I'm not sure if there is a better way to re-index everything? Also, since there isn't a build tool recognized for the tests, this fails anyways. Any thoughts on whether this is the best thing to do here or if there is a better alternative?

Reply:
We already do something similar when reconnecting automatically in the `MetalsLAnguageServer.scala` - not sure if helpful:
```scala
  session.mainConnection.onReconnection { newMainConn =>
              val updSession = session.copy(main = newMainConn)
              connectToNewBuildServer(updSession)
                .flatMap(compileAllOpenFiles)
                .ignoreValue
            }
```